test/multiple-nodes.spec.js

```diff
@@ -100,10 +96,10 @@ describe('multiple nodes', () => {
     await dhts[4].put(key, Buffer.from('world4'))
 
     const res = await Promise.all([
-      dhts[3].get(key, { maxTimeout: 2000 }),
```

Comment:
I dont't know as this was passing before. 

It was getting stored locally `world3` (after put), which was the first result obtained from `getMany`. Then, we use the `selector: () => 0` and the first record was obtained